torchmultimodal/models/omnivore.py

```diff
@@ -89,7 +130,7 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return x
 
 
-def _omnivore_swin_t_encoder() -> SwinTransformer3d:
+def omnivore_swin_t_encoder(pretrained=False, progress=True) -> SwinTransformer3d:
```

Comment:
do u really need to expose the progress param here?

Reply:
Actually no need to expose, I just follow [torchvision style](https://github.com/pytorch/vision/blob/main/torchvision/models/vision_transformer.py#L699) but no strong opinion on this.

If you think not exposing is better, then I will remove it. One question though, if we are not exposing, should we do `progress=False` or `progress=True` on the underlying call?

Reply:
I am pro exposing them as the callsite may not want to default `pretrained=False`.

Reply:
I think overall no one has strong opinion on removing `progress=False` hence I will resolve this